### ISSUE TITLE
Setting Websocket Url

### ISSUE THREAD

**Issue by D0miH** [jstnhuang/ros-rviz#22]
Hello,

I encapsulated the ros-rviz and the ros-websocket element in my own Polymer component.
Within the ```connectedCallback()``` I am trying to set the url which I passed to my component as a property.

```JavaScript
connectedCallback() {
    super.connectedCallback();
    let rviz = this.$.rviz;
    rviz.config = {
        "globalOptions": {
            "url": "ws://" + this.url + ":9090",
            "background": "#111111",
            "colladaLoader": "collada2",
            "colladaServer": "http://" + this.url + ":8001/",
            "fixedFrame": "/base_link",
            "videoServer": "http://" + this.url + ":9999"
        }
    }
    this.$.websocket.connect();
}
```

Setting the url for the colladaServer and the videoServer is working.
But the Websocket URL is not updating. When I check the global options in the sidebar, the Websocket URL is unchanged.

<img width="288" alt="screen shot 2018-05-13 at 09 24 47" src="https://user-images.githubusercontent.com/36657497/39964940-b13848ee-568f-11e8-9184-34d3b47b5704.png">

I was wondering if you might have any idea what the problem could be.

Thanks,
Dominik

**Comment by jstnhuang** [comment]
Hi Dominik,

It probably has to do with the way Polymer handles changes to objects. I don't think ros-rviz monitors changes to the config object itself. So, you might need to add a line of code like before `this.$.websocket.connect()`.

`rviz.set('config.globalOptions.url', 'ws://' + this.url + ':9090');`

**Comment by D0miH** [comment]
Hi,

thanks for your fast response. Your suggestion didn't fix my issue. 
Apparently the correct url was used. But the updated url was not dispayed in the global options.

I made a pull request #23 which should fix the issue.

Thanks again for your help,
Dominik

**Comment by jstnhuang** [comment]
Thanks very much! Closed by #23
